test/e2e/iam/project_settings_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || (iam && atlas)
+
+package iam_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/mongodb/mongodb-atlas-cli/test/e2e"
+	"github.com/stretchr/testify/require"
+	exec "golang.org/x/sys/execabs"
+)
+
+func TestProjectSettings(t *testing.T) {
+	cliPath, err := e2e.Bin()
+	require.NoError(t, err)
+
+	n, err := e2e.RandInt(1000)
+	require.NoError(t, err)
+
+	projectName := fmt.Sprintf("e2e-proj-%v", n)
+	projectID, err := createProject(projectName)
+	require.NoError(t, err)
+
+	defer func() {
+		if e := deleteProject(projectID); e != nil {
+			t.Errorf("error deleting project: %v", e)
+		}
+	}()
+
+	t.Run("Describe", func(t *testing.T) {
```

Comment:
are we able to test the describe command without creating a setting first? what is the output of decribe (text and json) when you have no settings?

Reply:
FYI: For this command, it's not required to create anything: everything is already there when we create the project. Resolved on a Zoom call.